tools/ambreModelInterpolation4DRBF.py

```diff
@@ -0,0 +1,181 @@
+import numpy as np
+import pandas as pd
+from astropy.io import fits
+from astropy.table import Table
+from astropy.io import ascii
+from scipy.interpolate import Rbf
+import parallel
+
+
+class AmbreInterpolation: 
+    """ Interpolate a model spectrum along four model parameters using scipy.interpolate.Rbf.
+        `interpolate` method returns a `numpy.ndarray` of an interpolated spectrum.
+        This script calls `parallel.py`.
+
+    Parameters
+    ----------
+    teff : Effective temperature (Teff [K]) at an interpolation. type = float
```

Comment:
Should be formatted as:
``
teff : `float`
    Effective temperature (T_eff [K]) at an interpolation point
``

And the same for the other parameters.
See https://developer.lsst.io/python/numpydoc.html#py-docstring-basics for more info/examples

Reply:
Different code was reviewed.